test/util/test_case.hpp

```diff
@@ -38,6 +38,7 @@ namespace ngraph
             {
             }
 
+            void set_tolerance(int tolerance_bits) { m_tolerance_bits = tolerance_bits; }
```

Comment:
I'm wondering why not to move this _tolerance bits_ value to `run` function as its input parameter?

Reply:
The `run` function would end up with a lot of different signatures, as different comparator functions will have different parameters, all of them optional. Better to configure class and leave run without parameters.